awx/main/tests/unit/test_validators.py

```diff
@@ -157,3 +158,13 @@ def test_valid_vars(var_str):
 def test_invalid_vars(var_str):
     with pytest.raises(RestValidationError):
         vars_validate_or_raise(var_str)
+
+
+def test_valid_system_path():
+    validate_system_path("/var")
```

Comment:
This probably needs an assertion to be a meaningful test.  Otherwise, it's just code that runs.  Something like:

```python
assert validate_system_path("/var") is True
```